packages/web-components/src/tree/tree.stories.ts

```diff
@@ -83,6 +83,21 @@ const storyTemplate = html<TreeStoryArgs>`
     </fluent-tree-item>
     <fluent-tree-item expanded>
       Item 2
+      <svg
+        slot="aside"
+        fill="red"
```

Comment:
`fill="red"`? is that correct?

Reply:
It's slotted content to show a warning. We do assign `color` to slots in the styles but it's just for a demosration of the `aside` slot